core/src/test/scala/org/http4s/blaze/pipeline/stages/SSLStageSuite.scala

```diff
@@ -61,7 +61,9 @@ class SSLStageSuite extends BlazeTestSuite {
       } yield ()
     }
 
-    test(testSuitePrefix + " should split large buffers") {
+    test((testSuitePrefix + " should split large buffers").flaky) {
```

Comment:
It makes me whaaaat
```scala
test(testSuitePrefix + " should split large buffers".flaky) {} // compiles but doesn't work properly
```

Reply:
though nothing new, silly type-weakness of string concatenation

Reply:
Oh, good find.  I always thought it was odd that operator went on the String in the first place.